index.js

```diff
@@ -0,0 +1,3 @@
+window.Ps = require('perfect-scrollbar');
```

Comment:
We're using CommonJS, but we're just not CommonJS enough to avoid exposing stuff on `window` :(